auto_update_tests.py

```diff
@@ -2,6 +2,8 @@
 
 import os, sys, subprocess, difflib
 
+os.environ['LD_LIBRARY_PATH'] = 'lib' # find our dynamic libraries
```

Comment:
This is probably a better solution:
https://cmake.org/Wiki/CMake_RPATH_handling